README.md

```diff
@@ -64,6 +64,14 @@ You can also choose another destination directory and release version for the in
 curl -sSfL https://raw.githubusercontent.com/anchore/grype/main/install.sh | sh -s -- -b <DESTINATION_DIR> <RELEASE_VERSION>
 ```
 
+### Compiling from the source
+
+```bash
+# git clone https://github.com/anchore/grype
+# cd grype
+# go build -o grype
```

Comment:
I agree with bumping this down in the readme as it should be the least recommended official install approach. 

I also agree with @kzantow about preferring `go install` over `go build` for this particular case. If this is intended for devs then we should really expand the `DEVELOPING.md` guidance instead of the installation section